### DISCUSSION TITLE
SyntaxTrivia needs a FunFacts property

### DISCUSSION THREAD

**Discussion by XAML-Knight** [dotnet/roslyn#58072] [General]
## Proposed API

```diff
namespace Microsoft.CodeAnalysis
{
     public class SyntaxTrivia : IEquatable<SyntaxTrivia>
     {
+        public List<string> FunFacts{ get; }
     }
```

**Comment by ghost** [comment]
I couldn't figure out the best area label to add to this issue. If you have write-permissions please help me learn by adding exactly one area label.

**Comment by CyrusNajmabadi** [comment]
I'm not sure if this proposal is serious or not.  If serious, please add more information as to what you're asking for @XAML-Knight .  If not serious, please close.

**Comment by jmarolf** [comment]
If not serious then I agree we should maybe put a link to imdb in there. 

Slightly related, I do wish we stopped calling them "code comments" in the first place. Hopefully "code trivia" catches on. like:

```csharp
// Did you know this is the longest method name in all of roslyn?
```
https://github.com/dotnet/roslyn/blob/8a92d8abaffdebe23ff777f692ee0cf26103fdf5/src/EditorFeatures/Test2/IntelliSense/CSharpCompletionCommandHandlerTests.vb#L2966

**Reply by Youssef1313** [reply]
Nope, the longest method name is:

https://github.com/dotnet/roslyn/blob/8a92d8abaffdebe23ff777f692ee0cf26103fdf5/src/EditorFeatures/VisualBasicTest/Diagnostics/GenerateType/GenerateTypeTests_Dialog.vb#L870

[SharpLab](https://sharplab.io/#v2:EYLgtghglgdgNAFxAJwK4wD4AEBMBGAWAChis8BOACgCIAVAUwGcEARAewDk2EBJGR+sgQBhNgBs2MAMptaACwgIAqgOTCIMUWAAOY+gnoB5VAgCCANQjIoEYHo4Qw9eYvZMuI1MmT0YCMQCeAKIAHlDMAPo8QRHy9MIBaIzqAtQAdAAyvgDmCHIAlADcpBQ0AOK+gorOAdr0fAhsUo70GRrZqBDZ9CxQAGZ9gr4IAArIbABW9ADGCBz0AO4AYlB6EUviACaCjBFcKrDZux5SUDr2LRFjkzMIAEr0gz4w0/TpWTC5BYVAA==) :)

**Reply by jmarolf** [reply]
Dang you `NotSimpleName` for coming in and pushing it over the top

**Reply by Xyncgas** [reply]
The terror
